devito/interfaces.py

```diff
@@ -2,6 +2,10 @@
 from sympy import IndexedBase, Function, as_finite_diff
 from sympy.abc import x, y, z, t, h, s
 from tools import aligned
+import os
+from signal import *
```

Comment:
`*` imports are not great, since they clobber the namespace. Can we please be explicit?